packages/core/src/arbitrary_value.rs

```diff
@@ -65,7 +65,7 @@ impl<'a> std::fmt::Display for AttributeValue<'a> {
             AttributeValue::Vec4Float(_, _, _, _) => todo!(),
             AttributeValue::Vec4Int(_, _, _, _) => todo!(),
             AttributeValue::Vec4Uint(_, _, _, _) => todo!(),
-            AttributeValue::Bytes(_) => todo!(),
+            AttributeValue::Bytes(a) => write!(f, "{:?}", a),
```

Comment:
`:?` is for Debug formatting, which is usually only used for developer-facing messages, traces, etc. I think it would be more correct to just use Display formatting as above, if possible.

Reply:
I thought the same but It didn't let me, I'll take a look again later

Reply:
I couldn't find an alternative way. Any suggestions? This works but I agree that using the `Debug` trait for this is weird.

Reply:
You're right, I've given this some more thought.

I suspect the issue here is that we shouldn't be implementing `Display` in the first place, since that's not what it's used for. I dug a bit through the codebase, and the only usage I could find was rendering elements to HTML in `TextRenderer::html_render`. It seems to me like the best way here would have been to serialize the HTML with Serde.

Anyway, using Debug here as a hack is fine with me now. I don't really like this code but that's not your fault :) and I have no better suggestions.